lms/djangoapps/courseware/courses.py

```diff
@@ -470,3 +465,79 @@ def get_problems_in_section(section):
                     problem_descriptors[unicode(component.location)] = component
 
     return problem_descriptors
+
+
+def get_current_child(xmodule, min_depth=None, requested_child=None):
```

Comment:
Yes, I added the changes in code and didn't comment. :-) I'll make sure I've replied to all of your comments now.